closed/adds/jdk/src/solaris/native/jdk/crypto/jniprovider/NativeCrypto_md.c

```diff
@@ -46,7 +51,8 @@ void * load_crypto_library() {
     "libcrypto.1.1.dylib",
     "libcrypto.1.0.0.dylib",
     "libcrypto.dylib"
-#else
+#elif defined(UNIX)
```

Comment:
This needs to be left alone: I don't see how `UNIX` could be defined.

Reply:
changed to __Linux__